DAPM/DAPM.AccessControlService/Controllers/OrganizationController.cs

```diff
@@ -0,0 +1,42 @@
+using DAPM.AccessControlService.Core.Services.Abstractions;
+using Microsoft.AspNetCore.Mvc;
+using RabbitMQLibrary.Models.AccessControl;
+
+namespace DAPM.AccessControlService.Controllers;
+
+[ApiController]
+[Route("api/access-control/organization")]
+public class OrganizationController(IOrganizationService organizationService) : ControllerBase
+{
+    [HttpPost]
+    [Route("add-user-organization")]
```

Comment:
We could also use the ["action"] in the http attribute to remove this line and add automatic route names from the controller methods 😎

Reply:
For me that it is bad practise because someone can easily change or refactor the name of the method and then the endpoint is changed  so always sticking with hardcoded paths for apis.